sky/clouds/service_catalog/kubernetes_catalog.py

```diff
@@ -26,3 +29,65 @@ def get_image_id_from_tag(tag: str, region: Optional[str]) -> Optional[str]:
 def is_image_tag_valid(tag: str, region: Optional[str]) -> bool:
     """Returns whether the image tag is valid."""
     return common.is_image_tag_valid_impl(_image_df, tag, region)
+
+
+def list_accelerators(
+        gpus_only: bool,
+        name_filter: Optional[str],
+        region_filter: Optional[str],
+        quantity_filter: Optional[int],
+        case_sensitive: bool = True
+) -> Dict[str, List[common.InstanceTypeInfo]]:
+    has_gpu = kubernetes_utils.detect_gpu_resource()
+    if not has_gpu:
+        return {}
+
+    label_formatter, _ = kubernetes_utils.detect_gpu_label_formatter()
+    if not label_formatter:
+        return {}
+
+    accelerators: Set[Tuple[str, int]] = set()
+    key = label_formatter.get_label_key()
+    nodes = kubernetes_utils.get_kubernetes_nodes()
+    for node in nodes:
+        if key in node.metadata.labels:
+            accelerator_name = label_formatter.get_accelerator_from_label_value(
+                node.metadata.labels.get(key))
+            accelerator_count = int(
```

Comment:
We currently return just the total count of GPUs available, but should we instead list the possible GPU counts a task can request? `sky show-gpus` is intended to show the possible quantities a task can request. For example:

A GKE cluster with 1 node with 4xT4 GPUs:
```
COMMON_GPU  AVAILABLE_QUANTITIES
T4          4
```
Because a task can request a subset of these 4 GPUs too, we should probably return:
```
COMMON_GPU  AVAILABLE_QUANTITIES
T4          1, 2, 3, 4
```

Reply:
Makes sense, I will update.

Reply:
Done in https://github.com/skypilot-org/skypilot/pull/2638/commits/f6b4ad6f03ff2e19e2b372eb1526444b5d1588a5